docs/DeveloperGuide.adoc

```diff
@@ -218,6 +218,46 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+// tag::removeTag[]
```

Comment:
Entire section that you have written has the same spacing issue that was prevalent throughout your User Guide (in another PR) as well. Remember that this is a guide, not a program, so don't break the line casually as it will change the display. Check by viewing the HTML version of the file